azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/flowtriggerspage.vm

```diff
@@ -130,6 +163,20 @@
                 </div>
               </div>
 
+              <td>${trigger.isPaused()}</td>
```

Comment:
Synced offline. We should also have a "pause" button inside Project Flow page.

Reply:
add it as todo.